fcq/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+# Create your models here.
+
+# Print helpers
+print_format = "{:<15}" * 2
+
+
+
+class Teacher(models.Model):
+	firstName = models.CharField(max_length=50)
+	lastName = models.CharField(max_length=50)
+	mainDepartment = models.CharField(max_length=50)
+	numClasses = models.IntegerField()
+	avgClassSize = models.IntegerField()
+	avgInstRating = models.FloatField()
+	avgCourseRating = models.FloatField()
+	avgChallenge = models.FloatField()
+	courseList = ArrayField(models.CharField(max_length=50),)
+	timesCourseTaught = ArrayField(models.IntegerField(),)
+	courseRating = ArrayField(models.FloatField(),)
+	courseInstRating = ArrayField(models.FloatField(),)
+	courseChallenge = ArrayField(models.FloatField(),)
+	classIndex = ArrayField(models.IntegerField(),)
+
+
+
+class FCQ(models.Model):
```

Comment:
FCQ model needs foreign keys to professor and class

Reply:
I don't see a better way of initializing those values than string comparison with the professor's name in the csv, and using the class department/code (i.e. "CSCI 1300")